tests/test_hls_report.py

```diff
@@ -0,0 +1,207 @@
+import heterocl as hcl
+import numpy as np
+import re
+import json
+import xmltodict
+import pathlib
+import imageio
+
+def sobel():
```

Comment:
We should import this from the example folder later on when `sobel` is tested. Maybe add a `TODO` here in case we forget?